test/service/test_response.py

```diff
@@ -1383,6 +1383,24 @@ def test_filter_by_projectId(self):
                     for term in response_json['termFacets']['project']['terms']:
                         self.assertEqual(term['projectId'], [test_data['id']])
 
+    def test_filter_by_contentDescription(self):
+        url = self.base_url + "/index/files"
+        params = self._params(size=3,
+                              filters={'contentDescription': {'is': ['RNA sequence']}},
+                              sort='fileName',
+                              order='asc')
+        response = requests.get(url, params=params)
+        response.raise_for_status()
+        response_json = response.json()
```

Comment:
```suggestion
        response = response.json()
```
I think you could reuse the response variable here but it is a styling preference so up to you.

Reply:
There seems to be a precedent for the `response_json` at least in `test_response.py`.
```
(.venv) daniel@Crispin ~/repo/azul1/test $ grep -r "= response.json()" .
./service/test_repository_proxy.py:                                        response = response.json()
./service/test_pagination.py:        json_response = response.json()
./service/test_pagination.py:        json_response = response.json()
./service/test_pagination.py:        json_response = response.json()
./service/test_pagination.py:        json_response_second = response.json()
./service/test_pagination.py:        json_response = response.json()
./service/test_pagination.py:        json_response_second = response.json()
./service/test_pagination.py:        json_response_third = response.json()
./service/test_pagination.py:                json_response = response.json()
./service/test_pagination.py:                json_response_second = response.json()
./service/test_pagination.py:                json_response_first = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:                    response_json = response.json()
./service/test_response.py:        response_json = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:        response = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:                    response = response.json()
./service/test_response.py:        response_json = response.json()
./service/test_response.py:        response_json = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:        response_json = response.json()
./service/test_response.py:        response_json = response.json()
./service/test_response.py:        response_json = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:        response_json = response.json()
./service/test_response.py:                response_json = response.json()
./service/test_response.py:        response_json = response.json()
./service/test_response.py:        summary_object = response.json()
./service/test_response.py:                summary_object = response.json()
./service/test_manifest_async.py:                            response = response.json()
./service/test_manifest.py:        hits = response.json()['hits']
./service/test_manifest.py:                response_json = response.json()
./service/test_request_validation.py:        actual_field_order = response.json()['order']
./service/test_request_validation.py:            response = response.json()
./service/test_repository_specimen.py:        response_json = response.json()
./integration_test.py:            body = response.json()
./integration_test.py:        spec = response.json()

(.venv) daniel@Crispin ~/repo/azul1/test $ grep -r " response_json = response.json()" . | wc -l
      22
(.venv) daniel@Crispin ~/repo/azul1/test $ grep -r " json_response = response.json()" . | wc -l
       5
(.venv) daniel@Crispin ~/repo/azul1/test $ grep -r " response = response.json()" . | wc -l
       5

```